bosh-setup/azuredeploy.json

```diff
@@ -113,10 +113,10 @@
 
     "baseUriAzureCloud": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/bosh-setup/",
```

Comment:
We agree there are a set of users that don't need to change the parameter value - that's why the BP details an approach that provides a default value for those users.  That same approach also works for other consumers of the repo...